swig/swigResqml2Include.i

```diff
@@ -4850,6 +4850,16 @@ namespace RESQML2_NS
 		WellboreMarkerFrameRepresentation const * getWellboreMarkerFrameRepresentation() const;
 		BoundaryFeatureInterpretation* getBoundaryFeatureInterpretation() const;
 		void setBoundaryFeatureInterpretation(BoundaryFeatureInterpretation* interp);
+		WITSML2_0_NS::WellboreMarker* getWitsmlWellboreMarker() const;
```

Comment:
Copying documentation in swig interface is now useful since SWIG version4 (I think only for java wrapper)